Update runbooks based on downstream wording (#240)
kubevirt/monitoring: docs/runbooks/CDINoDefaultStorageClass.md

## Meaning

This alert fires when there is no default (Kubernetes or KubeVirt) storage
class, and a data volume is pending for one.

A default KubeVirt storage class has precedence over a default Kubernetes
storage class for creating a VirtualMachine disk image.

## Impact

> If there is no default (Kubernetes or KubeVirt) storage class, a data volume
> that requests a default storage class (storage class not explicitly specified)
> will be pending for one.

Reviewer: Suggestion:
```suggestion
If there is no default Kubernetes or KubeVirt storage class, a data volume
that does not have a specified storage class remains in a "pending" state.
```

Author: updated

## Diagnosis

1. Check for a default Kubernetes storage class by running the following
command:

```bash
$ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubernetes.io/is-default-class"=="true")|.name'
```

2. Check for a default KubeVirt storage class by running the following command:

```bash
$ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name'
```

3. To set the default Kubernetes storage class if needed:

```bash
$ kubectl patch storageclass <storage-class-name> -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
```

4. To set the default KubeVirt storage class if needed:

```bash
$ kubectl patch storageclass <storage-class-name> -p '{"metadata": {"annotations":{"storageclass.kubevirt.io/is-default-virt-class":"true"}}}'

> ```

Reviewer: I would remove these lines about setting the default storage class because they are covered in the Mitigation section.

Author: removed
